src/app/content/highlights/components/Card.tsx

```diff
@@ -52,7 +52,7 @@ const Card = (props: CardProps) => {
   const annotation = props.data && props.data.annotation;
   const element = React.useRef<HTMLElement>(null);
   const [editing, setEditing] = React.useState<boolean>(!annotation);
-  const locationFilters = useSelector(selectHighlights.highlightLocationFilters);
+  const locationFilters = useSelector(contentSelect.highlightLocationFilters);
```

Comment:
this selector i think should still be in `selectHighlights`. its currently true that both MH and SG want to have the same chapter filter options, but it wouldn't be unreasonable for that to change and for instance MH gets pages and SG doesn't, then we'd want the locationFIlters to be different for each one. in that case the pagination logic would still work i think, just with different ids passed in.

actually we will probably want to have them be different before launch because SG will not have content in things like the index or preface, so we will probably not have non-chapter options in there the way MH does.

for now, since the underlying logic of the locationFilter selector will be the same for both, just define the selectors separately but delegate the implementation to some shared util function